sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/models/Document.java

```diff
@@ -33,13 +33,16 @@ public Document(T document, String partition, String id) {
         this.document = document;
     }
 
-    /**
-     * Create document from error.
-     *
-     * @param exception Error when retrieving the document.
-     */
-    public Document(Exception exception) {
-        this.documentError = new DocumentError(exception);
+    public Document(T document, String partition, String id, String eTag, long timestamp) {
+        this(document, partition, id);
+        this.eTag = eTag;
+        this.timestamp = timestamp;
+        this.document = document;
+    }
+
+    public Document(Exception dError)
```

Comment:
Rename dError to exception

Reply:
I thought it's renamed in my previous pr... what happened :thinking: